### ISSUE TITLE
Exchange Rates

### ISSUE THREAD

**Issue by rufuspollock** [datasets/awesome-data#15]
http://datahub.io/dataset/exchange-rates

**Comment by rufuspollock** [comment]
@Mikanebu can you look into this and report back on what it would take to get this up - i think this would be really useful.

**Comment by Mikanebu** [comment]
@rufuspollock Sure, I will report you soon

**Comment by Mikanebu** [comment]
This would be a huge dataset with several resoures:

**Comment by Mikanebu** [comment]
@rufuspollock This will be a huge dataset with 2 resources:
* `euro-foreign-exchanges-rates` 
  * from 1991 to nowadays.
  * regularity: daily
  * 41 columns(each column new currency)
* Foreign exchange rates from US federal reserve(daily and annually)

**Comment by Mikanebu** [comment]
@rufuspollock This will be a huge dataset with 2 resources:

* `euro-foreign-exchanges-rates`
  * from 1991 to nowadays.
  * regularity: daily
  * 41 columns(each column for foreign currency)
* `Foreign exchange rates from US federal reserve(daily and annually)`
  * probably we only need daily or we can split it out into 2 resources
  * 3 columns(Date,Rate,Currency)
* script is well written and needs small refactoring https://github.com/datasets/exchange-rates-usd/blob/master/run.py

Estimate time: 2hours

**Comment by rufuspollock** [comment]
@Mikanebu let's start with US Federal Reserve data. Do we have granularities other than daily (and what's the time coverage)?

Also can we automate this?

**Comment by Mikanebu** [comment]
@rufuspollock Yes, annually. Time coverage since 1971. We cannot automate US Federal Reserve data, it is bit complicated(zip with 148 csv files,plus some logic to merge them), but we can automate `euro-foreign-exchanges-rates`. So, I suggest having 2 different datasets for `EURO` and `USD`.

**Comment by anuveyatsu** [comment]
We could work on this dataset and improve to include more countries. Ideally, we could cover all countries from PPP dataset - https://github.com/datasets/ppp
